armi/physics/neutronics/tests/test_energyGroups.py

```diff
@@ -53,3 +53,16 @@ def test_consistenciesBetweenGroupStructureAndGroupStructureType(self):
                     energyGroups.getGroupStructure(groupStructureType)
                 ),
             )
+
+    def test_getFastFluxGroupCutoff(self):
+        """Test ability to get the ARMI energy group index contained in energy threshold.
+
+        .. test:: Return the energy group index which contains a given energy threshold.
+            :id: T_ARMI_EG_FE
+            :tests: R_ARMI_EG_FE
+        """
+        group, frac = energyGroups.getFastFluxGroupCutoff(
+            [100002, 100001, 100000, 99999, 0]
+        )
+
+        self.assertListEqual([group, frac], [2, 0])
```

Comment:
No action needed, @bdlafleur just venting

1. I wish `getFastFluxGroupCutoff` had a better docstring with the `parameters` and `returns` fields
2. `getFastFluxGroupCutoff` is unused in ARMI and only used in a downstream project. I would advocate we move it to said project... That way that project can maintain the requirement, implementation, and testing.